package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "dyngoose",
-  "version": "3.1.3",
+  "name": "@hixme/dyngoose",
+  "version": "3.2.0",
```

Comment:
Do not need to manually bump the version for release, this is done automatically from the CI release process.